app/src/components/mainWindow.ts

```diff
@@ -72,6 +72,16 @@ function setProxyRules(browserWindow: BrowserWindow, proxyRules): void {
   });
 }
 
+export function saveAppArgs(appArgs, filePath) {
```

Comment:
1. `appArgs` → `newAppArgs: any`, pls
2. Why make `filePath` an argument? Why not make `saveAppArgs` always write at `APP_ARGS_FILE_PATH`, so that the caller is only concerned with passing the new app args?